features/handled_exception.feature

```diff
@@ -11,6 +11,17 @@ Scenario: Test handled Kotlin Exception
     And the exception "message" equals "HandledExceptionScenario"
     And the payload field "events.0.device.cpuAbi" is a non-empty array for request 0
 
+Scenario: Report a handled exception without a message
```

Comment:
This scenario won't be run against browserstack as it's in the legacy mazerunner directory. It'd be worth discussing with @Cawllec whether we need to write new scenarios to target both browserstack _and_ the local emulator.